splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/VTIOperationIT.java

```diff
@@ -182,6 +183,18 @@ public void testVTIConversion() throws Exception {
         }
         Assert.assertEquals(5, count);
     }
+    //
+    @Test
+    public void testORCFileVTI() throws Exception {
+        String location = getResourceDirectory()+"orcVtiFile.in";
+        String sql = String.format("select * from new com.splicemachine.derby.vti.SpliceORCFileVTI('%s') as b (id bigint,name varchar(128),active char(1), start_dt date, update_dt timestamp )", location);
+        ResultSet rs = spliceClassWatcher.executeQuery(sql);
```

Comment:
Resource leak: You aren't closing the ResultSet. In tests it's not overly critical (we have a few backups in case stuff doesn't close cleanly), but we still should follow good practices and close what we open.